token/core/zkatdlog/crypto/common/schnorr.go

```diff
@@ -10,67 +10,116 @@ import (
 	"github.com/pkg/errors"
 )
 
-// Struct for Schnorr proofs
+// SchnorrProof carries a ZKP for statement (w_1, ..., w_n): Com = \prod_{i=1}^n P_i^w_i
```

Comment:
do you mean to include a Makefile entry to pass it over the code?

Reply:
I wouldn't go that far, I think we can just take a comment snippet, apply mathfmt on it, and copy the text...

Reply:
Also, it's not perfect. It has things it cannot handle, or at least back in the day it couldn't...  maybe they improved it since. 

https://github.com/SmartBFT-Go/randomcommittees/blob/master/internal/pvss.go#L311
https://github.com/SmartBFT-Go/randomcommittees/blob/master/internal/pvss.go#L43

Reply:
I just tried it, it does not work properly for power or indices.